### DISCUSSION TITLE
To view details about a check open in browser flow is very broken

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#46851] [Mobile] [answered]
GitHub Mobile beta

## steps
1. Browse to this PR (using GitHub Mobile)   https://github.com/grafana/grafana/pull/63164
2. Expand checks
3. Tap the backport check failed item
4. See:
    ![Screenshot_20230209-075944.png](https://user-images.githubusercontent.com/2119212/217819767-835ebcd0-3f53-4126-a64e-513f28b042ef.png)
   > Backport Check
   > To view details of this job open it in your browser to view it on the web.
   > [ OPEN IN BROWSER ]
5. Get very confused by the text (more later)
6. Click the share button at the top to see where I am
7. Click the button

## Expected results

1. Don't tell users to "open in browser" to "view on the web" – users understand that opening things in a browser will view them on the web
2. Clicking a button labeled "OPEN IN BROWSER" should open in my default web browser, otherwise the button should just be "open" or "view" or something.
3. Assuming the button and label are related, there's really no need for the text
4. Clicking such a button should actually show me details

## Actual results

1. One of these things takes me to https://github.com/grafana/grafana/blob/main/contribute/merge-pull-request.md
2. Another takes me to   https://github.com/grafana/grafana/blob/main/contribute/merge-pull-request.md#should-the-pull-request-be-backported
3. I'm taken to a web view but not my system default browser
4. None of these paths took me to the details of the job

https://user-images.githubusercontent.com/2119212/217821816-df79cdfb-2e4c-46da-8406-b84159e7e9e9.mp4

https://user-images.githubusercontent.com/2119212/217822402-4af2767b-2b17-491a-9ffa-27bf0aab5987.mp4

**Comment by Sumanprakash369** [comment]
👍

**Comment by taki-on** [comment, marked as answer]
Hi! Thank you for reporting the issue and the suggestions for the UI. I will pass your suggestions to our design team and we will discuss them :)

I checked out the pull request you linked above on **github.com** and found that _details_ for those failing checks open the markdown file. I guess those checks are not powered by GitHub Actions but probably are legacy GitHub checks. GitHub Mobile currently does not support such legacy checks, and for those checks we provide links to the web. Since GitHub Mobile jumps to the same destination I see on the web, I guess those checks are designed to do so. I want to kindly ask you to verify the checks one more time, and if you still think they are bugs on GitHub Mobile, please let us know!

Thanks!

**Reply by jsoref** [reply]
I'm not saying the destination is wrong, I'm saying the text about the buttons are wrong. It's totally ok for the destination to be a readme.

I think to fix the `Details` prose, you could change it from `To view details of this job` to `To view details generated by this job`.

I'm used to GitHub Actions where Details dumps me in a GitHub Actions Log (instead of a Summary, which frankly would be better 95% of the time).

**Reply by taki-on** [reply]
Aaah thank you for the clarification! I will pass your suggestion to our design folks! Thank you very much!
